src/components/search-result-list/SearchResultList.tsx

```diff
@@ -1,38 +1,44 @@
 import React, { memo } from "react";
-import { dataIsNotEmpty, getCoverTint } from "../../core/utils/helpers/general";
+import { isDataEmpty, getCoverTint } from "../../core/utils/helpers/general";
 import { Work } from "../../core/utils/types/entities";
 import SearchResultListItem from "./search-result-list-item/search-result-list-item";
 import SearchResultListItemSkeleton from "./search-result-list-item/search-result-list-item-skeleton";
 
 export interface SearchResultListProps {
   resultItems: Work[];
+  hitcount: number;
+  isLoading: boolean;
 }
 
-const SearchResultList: React.FC<SearchResultListProps> = ({ resultItems }) => {
-  const worksAreLoaded = dataIsNotEmpty(resultItems);
-
+const SearchResultList: React.FC<SearchResultListProps> = ({
+  resultItems,
+  hitcount,
+  isLoading
+}) => {
   return (
     <ul className="search-result-page__list my-32" data-cy="search-result-list">
       {/*
           Show skeleton search result items if no data is available yet.
           We'll show 5 items which should cover most screens.
         */}
-      {!worksAreLoaded &&
+      {isLoading &&
         [...Array(5)].map(() => (
           <li>
             <SearchResultListItemSkeleton />
           </li>
         ))}
-      {worksAreLoaded &&
-        resultItems.map((item, i) => (
-          <li key={item.workId}>
-            <SearchResultListItem
-              item={item}
-              coverTint={getCoverTint(i)}
-              resultNumber={i + 1}
-            />
-          </li>
-        ))}
+      {/* We fill out empty page without results a little bit to visually convey
+      to the user that the results are empty. */}
+      {(!hitcount || isDataEmpty(resultItems)) && <div className="my-112" />}
```

Comment:
Please reconsider adding this spacer here.

This mixes in considerations about spacing for empty results (which is not really a list) into a  component which is responsible for rendering a list. It also means that we have to pass information about the state of the search result (`hitcount` and `isLoading`) into the component.